### PR TITLE
lti generator, BUGFIX token has no .encode, lti uses correct secret.

### PR DESCRIPTION
**Pull request by BigDGW**
## Summary of changes
- Fixes token has no encode when entering to lti a second time.
- Fixes lti jwt encode uses our jwt secret
- .ADDED lti key secret couple generator

### CODE REVIEW

File: lti/lti_generator.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import os
+print('key')
+print(os.urandom(16).hex())
```
Comment (libre-man) [inline lti/lti_generator.py]: You can use the builtin `secrets` module to do this.

File: lti/lti_generator.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import secrets
+
+# Generates a new key and secret couple for the lti Oauth communication te key
+# and secrect are generated according to the guidelines given in
+# https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/
```
Comment (libre-man) [inline lti/lti_generator.py]: This is for oath2 servers, however LTI uses oath 1.

Comment (libre-man) [reply lti/lti_generator.py]: Not that it is incorrect or anything, just a bit weird.

Comment (BigDGW) [reply lti/lti_generator.py]: The key and secret specifications hasnt changed between Oauth1 and Oauth2. And thus on the offical site the link to the specifications for oauth1 key and secret also links to this link.

**Pull request merged**